html/user-profile.html

```diff
@@ -5,35 +5,79 @@
     <meta content="text/html; charset=UTF-8">
     <link rel="stylesheet" type="text/css" href="../css/style.css" />
 </head>
-<body>
-  <header>
-    <nav>
-      <ul>
-        <li><a href="user-profile.html">User Profile</a></li>
-        <li><a href="user-list.html">User List</a></li>
-        <li><a href="create-user.html">Create User</a></li>
-        <li><a href="edit-user.html">Edit User</a></li>
-      </ul>
-    </nav>
-  </header>
-  <table> 
-      <h3>User Profile</h3>
-    <tr>
-      <td>ID:</td>
-      <td>#0000</td>
-    </tr>
-    <tr>
-      <td>Name:</td>
-      <td>John Smith</td>
-    </tr>
-    <tr>
-      <td>Phone:</td>
-      <td>000-000-0000</td>
-    </tr>
-    <tr>
-      <td>Email:</td>
-      <td>john.smith@banno.com</td>
-    </tr>
-  </table>
-</body>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="user-profile.html">User Profile</a></li>
+          <li><a href="user-list.html">User List</a></li>
+          <li><a href="create-user.html">Create User</a></li>
+          <li><a href="edit-user.html">Edit User</a></li>
+        </ul>
+      </nav>
+    </header>
+    <h3>User Profile</h3>
+    <table> 
+      <tr>
+        <td>ID:</td>
+        <td>#0000</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>John Smith</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>000-576-5309</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>john.smith@banno.com</td>
+      </tr>
+      <tr>
+        <td>
+          <button onclick="cardExpandCollapse(this)">Collapse</button>
+        </td>
+      </tr>
+    </table>
+    <table> 
+      <tr>
+        <td>ID:</td>
+        <td>#0000</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>John Smith</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>000-576-5309</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>john.smith@banno.com</td>
+      </tr>
+      <tr>
+        <td>
+          <button onclick="cardExpandCollapse(this)">Collapse</button>
+        </td>
+      </tr>
+    </table>
+    <script>
+      function cardExpandCollapse(element) {
+        const button = element;
+        const collapses = element.parentNode.parentNode.parentNode.querySelectorAll(".expand");
```

Comment:
Specificity! I mean there may be some other selector that might let you jump to the parent element you want by targeting an id, but I can't think of/find it at the moment. I'll leave it to Kean on this one.

Reply:
I was thinking _maybe_ `document` or `window`? But that's a total stab in the dark.

Reply:
I tried doing that before, but by using `.childen()` it would make an array rather than let me grab the specific element i needed.

Reply:
and `element` acts as the `event` that's being targeted in this can. Should i change the name to `event`

Reply:
@phillnab nah, I'm not realizing how we deal with events in polymer isn't as close to vanilla as I had thought. what you have right here is right.